assets/dev/js/editor/document/hooks/data/document/elements/helper.js

```diff
@@ -14,16 +14,7 @@ export default class Helper {
 				/**
 				 * TODO: Try improve performance of using 'document/elements/create` instead of manual create.
 				 */
-				const createdContainer = container.view.addElement( model, { edit: false } ).getContainer();
-
-				$e.store.dispatch(
-					$e.store.get( 'document/elements' ).actions.create( {
-						documentId: elementor.documents.getCurrentId(),
-						parentId: container.id,
-						elements: [ createdContainer.model.toJSON() ],
-						index: options.at,
-					} ),
-				);
+				container.view.addChildModel( model );
```

Comment:
are you sure? seems like those are 2 different methods

Reply:
it's a revert of the original PR
but I'll change it back to this method, just to be safe